docs/ref/models/querysets.txt

```diff
@@ -2519,6 +2519,15 @@ The ``batch_size`` parameter controls how many objects are saved in a single
 query. The default is to update all objects in one batch, except for SQLite
 and Oracle which have restrictions on the number of variables used in a query.
 
+.. warning::  
+   When using MySQL, ``QuerySet.update()`` may execute two separate SQL queries  
```

Comment:
I think this would make more sense under `update()` than `bulk_update()`, given that `bulk_update()` already references delegating to `update()`.